### DISCUSSION TITLE
recommendation config

### DISCUSSION THREAD

**Discussion by Zhankin** [swar/Swar-Chia-Plot-Manager#100] [Q&A]
Could you get some recommendation:
based on CPU cores and threads, how many jobs, and threads per job need to config?
how to correct way to calculate stagger_minutes, max_concurrent_with_start_early?
or write you setup as example

**Comment by Amneamnius** [comment]
Checkout [https://thechiafarmer.com/2021/04/21/optimizing-plotters-in-windows/](url). Stagger eventually happens just by the cpu being loaded and some plots being delayed, but you can set initial staggering or extra staggering if you want. With Start Early is if you want to start any plots during the start early phase (4 by default), you should see if your drive has some space during this time to allow starting 1+ plots and update.

On a 1TB temp drive I run 3 phase 1, 3 concurrent, 4 with start early but can probably do 3/4/5. No stagger but it ends up with about 30m stagger due to my CPU.
